src/main/java/com/ericsson/ei/subscription/InformSubscriber.java

```diff
@@ -414,17 +415,18 @@ private String extractContextPath(String notificationMeta) throws MalformedURLEx
      * @param notificationMeta
      */
     private void saveMissedNotificationToDB(String aggregatedObject, String subscriptionName, String notificationMeta) {
-        String input = prepareMissedNotification(aggregatedObject, subscriptionName, notificationMeta);
-        LOGGER.debug("Input missed Notification document : " + input);
-        mongoDBHandler.createTTLIndex(missedNotificationDataBaseName, missedNotificationCollectionName, "Time",
-                ttlValue);
-        boolean output = mongoDBHandler.insertDocument(missedNotificationDataBaseName, missedNotificationCollectionName,
-                input);
-
-        if (!output) {
-            LOGGER.debug("Failed to insert the notification into database");
-        } else
+        try {
```

Comment:
This function does more then one thing, the try/catch also has parts in it that may not throw MongoWriteException that shouold be kept outside the try catch.

Reply:
I don't really understand how this can be more divided without losing the readability and understanding what the function does.

Reply:
"understanding what the function does." If you need to read the code to understand what the function does then the function name is wrong, cause the name should give that picture. If a function does more then the name indicates then the code should be split into more functions that indicates what each function does.

Reply:
I want to see what libs/function that is used an how, if that is hide in tiny functions , then its becomes very hard to review the code. I agee if the functions is very long or big, then its good to split it up to one or more functions. But this function is already very small and splitting it, it becomes very hard to review, because you need to jump back forth between the functions, if its many tiny functions than I can't follow the code. I would have hard to approve that kind of code change.

Reply:
I see the PRwas updated and I like it, if you have a try/catch you want to know what throws the error you catches. adding 20 lines of code just to know what libs was used is not a good idea, then functions should not be split to it´s extreme just because.